docs/index.html

```diff
@@ -6,22 +6,26 @@
     <script src="fourier-polygon.js"></script>
 </head>
 <body>
-    <textarea id="pathData" rows="6" cols="50"></textarea>
-    <button id="drawButton">Draw</button>
-    <h3>Sample Shapes:</h3>
-    <button id="basicButton">Basic square</button>
-    <button id="squareButton">Square</button>
-    <button id="devSquareButton">Square with outlying point</button>
-    <button id="horseButton">Horse</button>
-    <button id="lionButton">Lion</button>
-    
-    <div id="app"></div>
+	<textarea id="pathData" rows="6" cols="50"></textarea>
```

Comment:
Whitespace should be consistent across the file. Either use tabs everywhere, or spaces everywhere. Since the file was already spaces, I would recommend simply using that.